r/.Rbuildignore

```diff
@@ -31,3 +31,4 @@ STYLE.md
 ^revdep$
 ^vignettes$
 ^PACKAGING\.md$
+^__pycache__$
```

Comment:
The path has needs to be relative to the root dir.
```suggestion
^inst/__pycache__$
```

Reply:
(or alternatively we need to remove the line start/end markers `^ $` to ignore  pycache dirs regardless of location in the package. I prefer the more explicit version above though.